modules/gitops-argo-cd-properties.adoc

```diff
@@ -122,6 +162,7 @@ a|* `<autoscale>` - Server autoscale configuration options.
   * `<logLevel>` - The log level to be used by the Argo CD Server component. Valid options are  `debug`, `info`, `error`, and `warn`.
```

Comment:
Again I see the inconsistencies in capitalization. Previous lines have: `Argo CD server`, but here we have: `Argo CD Server`. These small inconsistencies should be addressed in the docs.

Reply:
Fixed.